code/processes/vwapsub.q

```diff
@@ -1,18 +1,20 @@
 \d .vwapsub // enter vwapsub namespace
 
-tickerplanttypes:@[value;`tickerplanttypes;`tickerplant]; // tickerplant types to subscribe to
-hdbtypes:@[value;`hdbtypes;`hdb];                         // hdbtypes to connect to
+tickerplanttypes:@[value;`tickerplanttypes;`tickerplant];                       // tickerplant types to subscribe to
+hdbtypes:@[value;`hdbtypes;`hdb];                                               // hdbtypes to connect to
 
 // datareplay settings
-realtime:@[value;`realtime;0b];                           // use realtime feed or datareplay. default is 0b (datareplay)
-replayinterval:@[value;`replayinterval;0D00:10:00.00];    // interval to run calcvwap at
-replaysts:@[value;`replaysts;2015.01.07D01:00:00.00];     // start time of data to retreive from hdb
-replayets:@[value;`replayets;2015.01.08D17:00:00.00];     // end time of data to retrieve from hdb
-
+realtime:@[value;`realtime;1b];                                                 // use realtime feed or datareplay. default is 0b (datareplay)
```

Comment:
this changed from `0b` default to `1b` default, was that intended?

Reply:
it doesn't matter much because the same variable is defined in settings/vwapsub.q as well and that's `0b` and overwrites the above. I will change it back though. I set it to 1 to test something.